🐛 [Bug] enabled_precisions needs to be removed from examples when use_explicit_typing is set

##  Bug Description
```

Traceback (most recent call last):
  File "/workspace/TensorRT/examples/dynamo/torch_compile_resnet_example.py", line 117, in <module>
    trt_model = torch_tensorrt.compile(model, **compile_spec)
  File "/opt/conda/envs/py310/lib/python3.10/site-packages/torch_tensorrt/_compile.py", line 311, in compile
    trt_graph_module = dynamo_compile(
  File "/opt/conda/envs/py310/lib/python3.10/site-packages/torch_tensorrt/dynamo/_compiler.py", line 637, in compile
    raise AssertionError(
AssertionError: use_explicit_typing was set to True, however found that enabled_precisions was also specified (saw: {torch.float16}, expected: dtype.f32, dtype.f4). enabled_precisions should not be used when use_explicit_typing=True

```
## To Reproduce

Steps to reproduce the behavior:

1.
2.
3.

<!-- If you have a code sample, error messages, stack traces, please provide it here as well -->

## Expected behavior

<!-- A clear and concise description of what you expected to happen. -->

## Environment

> Build information about Torch-TensorRT can be found by turning on debug messages

 - Torch-TensorRT Version (e.g. 1.0.0):
 - PyTorch Version (e.g. 1.0):
 - CPU Architecture:
 - OS (e.g., Linux):
 - How you installed PyTorch (`conda`, `pip`, `libtorch`, source):
 - Build command you used (if compiling from source):
 - Are you using local sources or building from archives:
 - Python version:
 - CUDA version:
 - GPU models and configuration:
 - Any other relevant information:

## Additional context

<!-- Add any other context about the problem here. -->

--- a/examples/dynamo/torch_compile_resnet_example.py
+++ b/examples/dynamo/torch_compile_resnet_example.py
@@ -48,6 +48,7 @@ optimized_model = torch_tensorrt.compile(
     model,
     ir="torch_compile",
     inputs=inputs,
+    use_explicit_typing=False,
     enabled_precisions=enabled_precisions,
     workspace_size=workspace_size,
     min_block_size=min_block_size,
@@ -86,6 +87,7 @@ optimized_model = torch_tensorrt.compile(
     model,
     ir="torch_compile",
     inputs=inputs_bs8,
+    use_explicit_typing=False,
     enabled_precisions=enabled_precisions,
     workspace_size=workspace_size,
     min_block_size=min_block_size,
@@ -111,6 +113,7 @@ compile_spec = {
             dtype=torch.half,
         )
     ],
+    "use_explicit_typing": False,
     "enabled_precisions": enabled_precisions,
     "ir": "dynamo",
 }

